src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/custom.py

```diff
@@ -270,6 +260,60 @@ def db_update(
 
 
 ###############################################
+#                sql dw                       #
+###############################################
+
+
+# Creates a datawarehouse. Wrapper function which uses the server location so that the user doesn't
+# need to specify location.
+def dw_create(
+        client,
+        database_name,
+        server_name,
+        resource_group_name,
+        **kwargs):
+
+    # Set edition
+    kwargs['edition'] = 'DataWarehouse'
```

Comment:
Can you reference the enum directly, in case the value changes in the future?  
```Python
kwags['edition'] = MyEnum.DataWarehouse.value # or omit .value to just use the enum type
```